extender/scheduler.go

```diff
@@ -19,6 +19,7 @@ func postOnly(next http.HandlerFunc) http.HandlerFunc {
 			log.Print("method Type not POST")
 			return
 		}
+
```

Comment:
Are these from gofmt or just preference? I'm wondering if different versions of gofmt might pick this up differently.

Reply:
These are indeed from the static scans with "everything on"